### ISSUE TITLE
crypto_rs README.md

### ISSUE THREAD

**Issue by mateolafalce** [CM-IV/crypto-rs#1]
Hello!, 
I have recently used this encryptor and I find it great. I come to propose something for the `readme.md`. I see that you show the program with a screen shot of the screen, do you think it's a good idea to make a demonstration of it via .gif, for a more dynamic experience? 
I am willing to contribute of course!

**Comment by CM-IV** [comment]
Hey there @mateolafalce,

I'm glad to hear that you like `crypto_rs`, as I enjoyed using `fltk-rs` in its implementation.

I added a screenshot in the `README.md` for each branch of the repo, as the different branches have different ways of integrating the age encryption library.  I'm certainly open to the idea of gifs being used to show a demonstration.

I should probably also re-write the `README.md`s on each branch with more information.
